src/init.cpp

```diff
@@ -358,6 +358,13 @@ void SetupServerArgs()
     gArgs.AddArg("-minimumchainwork=<hex>", strprintf("Minimum work assumed to exist on a valid chain in hex (default: %s, testnet: %s)", defaultChainParams->GetConsensus().nMinimumChainWork.GetHex(), testnetChainParams->GetConsensus().nMinimumChainWork.GetHex()), true, OptionsCategory::OPTIONS);
     gArgs.AddArg("-par=<n>", strprintf("Set the number of script verification threads (%u to %d, 0 = auto, <0 = leave that many cores free, default: %d)",
         -GetNumCores(), MAX_SCRIPTCHECK_THREADS, DEFAULT_SCRIPTCHECK_THREADS), false, OptionsCategory::OPTIONS);
+    gArgs.AddArg("-mitigatespectre", strprintf("Attempt to enable per process spectre mitigations (default: %u%s)", DEFAULT_MITIGATE_SPECTRE, (
+#ifdef ENABLE_WALLET
+    DEFAULT_MITIGATE_SPECTRE ? " or 0 if -disablewallet set" : "0"
+#else
+    strprintf("%u", DEFAULT_MITIGATE_SPECTRE)
```

Comment:
Am I misreading this code completely? This looks like it will print out `DEFAULT_MITIGATE_SPECTRE` twice.

Reply:
`DEFAULT_MITIGATE_SPECTRE` is somewhat redundant in `DEFAULT_MITIGATE_SPECTRE ? " or 0 if -disablewallet set" : "0"` since `DEFAULT_MITIGATE_SPECTRE` is controlled by `ENABLE_WALLET`, doesn't look like it would be printed twice.

Reply:
The first `0` comes from `%u` in the initial format line and `DEFAULT_MITIGATE_SPECTRE`. The second `0` comes from
```C++
    strprintf("%u", DEFAULT_MITIGATE_SPECTRE);
```
in the else case on line 365.

Reply:
ah, removed that line now